server/controllers/content.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable sort-keys */
+/* eslint-disable consistent-return */
+const Content = require("../models/Content");
+
+// Create and Save a new Content
+exports.create = (req, res) => {
+
```

Comment:
All through out this file and your code, destructor the `req.body` object for the keys that you want to use (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)